staking/types/validator.go

```diff
@@ -510,8 +541,18 @@ func UpdateValidatorFromEditMsg(validator *Validator, edit *EditValidator) error
 		}
 	}
 
-	if edit.Active != nil {
-		validator.Active = *edit.Active
+	switch validator.EPOSStatus {
+	case effective.Banned:
+		return errCannotChangeBannedTaint
+	default:
+		switch edit.EPOSStatus {
+		case effective.Active, effective.Inactive:
+			validator.EPOSStatus = edit.EPOSStatus
+		default:
+			return errors.Wrapf(
```

Comment:
When the value is 0. It should mean no-op (as in nil pointer before). Don't return error.